Automattic/newspack-plugin
# Info Buttons for Checkboxes

### All Submissions:






### Changes proposed in this Pull Request:

Add an optional `tooltip` attribute to the `CheckboxInput` component, which will render a Muriel Info icon to the right which with a Gutenberg Tooltip associated with it.

![](https://user-images.githubusercontent.com/7317227/56914259-801c6b80-6a68-11e9-9cca-5e8875d1815d.png)



Closes #24.

### How to test the changes in this Pull Request:

1.  Build  the components: `npm ci && npm run build:webpack`
2. In a test environment running the Newspack plugin, navigate to Newspack->Subscriptions.
3. Observe two checkbox components, the second of which has the Info icon.
4. Hover over the icon and observe a tooltip. 

### Other information:
